framework_tvb/tvb/core/adapters/constants.py

```diff
@@ -73,3 +73,8 @@
 
 ALL_TYPES = (TYPE_STR, TYPE_FLOAT, TYPE_INT, TYPE_UPLOAD, TYPE_BOOL, TYPE_DICT,
              TYPE_ARRAY, TYPE_SELECT, TYPE_MULTIPLE, TYPE_DYNAMIC, TYPE_LIST)
+
+ENCRYPTED_PASSWORD_NAME = 'encrypted_password.pem'
+ENCRYPTED_DATA_SUFFIX = '_encrypted'
+DECRYPTED_DATA_SUFFIX = '_decrypted'
```

Comment:
I know it was my suggestion to have the constants here, but now they only seem used inside `abcuploader.py`. Could they be better there ?

Reply:
Done.